ledger-service/http-json/src/main/scala/com/digitalasset/http/PackageService.scala

```diff
@@ -62,6 +62,17 @@ private class PackageService(
       val findIface = iface.Interface.findAstInterface(Function unlift lookupIf)
       diff.transform((_, iface) => iface resolveChoicesAndIgnoreUnresolvedChoices findIface)
     }
+
+    private[this] def appendAndResolveRetroactiveInterfaces(diff: PackageStore): PackageStore = {
```

Comment:
For new readers the `PackageStore` type alias is just another thing to lookup in the code.  Using `Map[String, Interface]` makes that lookup unnecessary.

Reply:
I guess this is more an issue with me reading through these diffs than with a type alias..